src/main/java/io/leonis/zosma/ipc/peripheral/component/Playstation.java

```diff
@@ -0,0 +1,35 @@
+package io.leonis.zosma.ipc.peripheral.component;
+
+/**
+ * The class Playstation.
+ *
+ * Describes the buttons on a playstation controller.
+ *
+ * @author Jeroen de Jong
+ */
+public class Playstation {
```

Comment:
`PlayStationController`?

Reply:
It's solely the playstation specific components though. But I think it's clearer if I make a package called `xbox` and `playstation` with the Supplier-classes.

Reply:
that would make more sense than having this class. what i was trying to say is that if these interfaces are describing components which are playstation specific, then there should be a `PlayStationController` class in which these interfaces are nested